{"name": "Login with wrong email", "status": "passed", "description": "Signing in with the following credentials email : kamelmohsenkamel & password: Kimo2010", "attachments": [{"name": "SUCCESS: Login failed with incorrect credentials", "source": "9dcc89e0-7ab5-47c2-b585-43149616b376-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "a1b4bdde-41dd-4627-a55c-3aa877921b3b-attachment.txt", "type": "text/plain"}], "start": 1586148113897, "stop": 1586148121645, "uuid": "a8932e25-2c0c-47d1-bcb4-deeb5101a83e", "historyId": "52536d47157233bdd508b7f9b78baaa3", "testCaseId": "2fde77b51224bed592a0c95ef44c32cb", "fullName": "Web_Testing.Tests.test_login.TestSignin#test_case_6", "labels": [{"name": "suite", "value": "Login Tests"}, {"name": "feature", "value": "Log in Tests"}, {"name": "severity", "value": "blocker"}, {"name": "story", "value": "Failing login"}, {"name": "subSuite", "value": "Login with wrong email"}, {"name": "parentSuite", "value": "End to End testing"}, {"name": "tag", "value": "Do"}, {"name": "tag", "value": "Login"}, {"name": "host", "value": "LAPTOP-8H6HECEC"}, {"name": "thread", "value": "19944-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Web_Testing.Tests.test_login"}]}